src/main/scala/edu/berkeley/ce/rockslicing/BlockVTK.scala

```diff
@@ -37,38 +36,38 @@ object BlockVTK {
     // vertical line running through this center.
     // Check if points are on opposite sides of the center. Points left of center will be before points
     // right of the center
-    if ((pointA._1 - center._1 < -NumericUtils.EPSILON) && (pointB._1 - center._1 >= NumericUtils.EPSILON)) {
+    if ((pointA(0) - center(0) < -NumericUtils.EPSILON) && (pointB(0) - center(0) >= NumericUtils.EPSILON)) {
       return true
     }
-    else if ((pointA._1 - center._1 >= NumericUtils.EPSILON) && (pointB._1 - center._1 < -NumericUtils.EPSILON)) {
+    else if ((pointA(0) - center(0) >= NumericUtils.EPSILON) && (pointB(0) - center(0) < -NumericUtils.EPSILON)) {
       return false
     }
 
     // Compares points that fall on the x = center._1 line.
-    if ((math.abs(pointA._1 - center._1) < NumericUtils.EPSILON) &&
-        (math.abs(pointB._1 - center._1) < NumericUtils.EPSILON)) {
+    if ((math.abs(pointA(0) - center(0)) < NumericUtils.EPSILON) &&
+        (math.abs(pointB(0) - center(0)) < NumericUtils.EPSILON)) {
       // Points furthest away from the center will be before points that are closer to the center
-      if ((pointA._2 - center._2 >= NumericUtils.EPSILON) || (pointB._2 - center._2 >= NumericUtils.EPSILON)) {
-        return pointA._2 > pointB._2
+      if ((pointA(1) - center(1) >= NumericUtils.EPSILON) || (pointB(1) - center(1) >= NumericUtils.EPSILON)) {
+        return pointA(1) > pointB(1)
       } else {
-        return pointB._2 > pointA._2
+        return pointB(1) > pointA(1)
       }      
     }
 
     // The cross product of vectors (pointA - center) and (pointB - center) in determinant form. Since it's
     // in 2-D we're only interested in the sign of the resulting z-vector.
-    val det = (pointA._1 - center._1) * (pointB._2 - center._2) -
-      (pointB._1 - center._1) * (pointA._2 - center._2)
+    val det = (pointA(0) - center(0)) * (pointB(1) - center(1)) -
+      (pointB(0) - center(0)) * (pointA(1) - center(1))
     // If resulting vector points in positive z-direction, pointA is before pointB
     if (det > NumericUtils.EPSILON) {
-      return true
+      true
     } else if (det < -NumericUtils.EPSILON) {
-      return false
+      false
     } else {
       // pointA and pointB are on the same line from the center, so check which one is closer to the center
-      val d1 = (pointA._1 - center._1) * (pointA._1 - center._1) + (pointA._2 - center._2) * (pointA._2 - center._2)
-      val d2 = (pointB._1 - center._1) * (pointB._1 - center._1) + (pointB._2 - center._2) * (pointB._2 - center._2)
-      return d1 > d2
+      val d1 = (pointA(0) - center(0)) * (pointA(0) - center(0)) + (pointA(1) - center(1)) * (pointA(1) - center(1))
+      val d2 = (pointB(0) - center(0)) * (pointB(0) - center(0)) + (pointB(1) - center(1)) * (pointB(1) - center(1))
+      d1 > d2
     }
```

Comment:
I see you changed your mind about explicitly putting `return`s here for readability :)